data/kalkulacka/senatni-2022/52.json

```diff
@@ -1059,25 +1059,27 @@
       "id": "senatni-2022-52-a-800084-28",
       "candidate_id": "senatni-2022-52-c-800084",
       "question_id": "senatni-2022-global-q-28",
-      "answer": "no"
+      "answer": "yes"
```

Comment:
Tady se mění odpovědi. To by se nemělo dít, ne? /cc @michalskop

Reply:
Ono to bere posledni odpoved od dane strany. Takze pokud odpovi novy clovek, tak se pouziji jine odpovedi.